autoload/neomake.vim

```diff
@@ -873,7 +875,9 @@ function! neomake#MakeHandler(job_id, data, event_type) abort
         if neomake#has_async_support()
             if !len(filter(copy(s:jobs), 'v:val.make_id == jobinfo.make_id'))
                 call neomake#utils#hook('NeomakeFinished', {
-                            \ 'file_mode': maker.file_mode})
+                            \ 'file_mode': maker.file_mode,
+                            \ 'bufnr': get(jobinfo, 'bufnr', -1),
```

Comment:
It is bad that there is no test that gets the -1.  Can it happen?  Otherwise we should remove the default from `get()`.
Do you feel like adding one?